ereuse_devicehub/resources/device/models.pyi

```diff
@@ -152,7 +152,7 @@ class Computer(DisplayMixin, Device):
         self.components = ...  # type: Set[Component]
         self.actions_parent = ...  # type: Set[e.Action]
         self.chassis = ...  # type: ComputerChassis
-        self.owner_address = ... # type: UUID
+        self.owner_id = ... # type: UUID
```

Comment:
in my opinion, this pull rquest should only be about tests and not involve other files such as models, views, schemas, etc

Reply:
Okay, fine with me. But if it is a necessary change for a test to pass correctly, in these cases I understand that we can make changes in the models, views, etc... To fix it!